doc/Overview/Reactive/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: Overview.Reactive.Feed
+---
+
+# MVUx Overview
+
+## What is MVU
+
+MVU stands for **M**odel-**V**iew-**U**pdate.  
+
+It utilizes a flow where the *model* is rendered on the *view* side, and once changed by request from user, gets *update*d.
+
+It enables leaving the actual model plain and immutable (ideally using records), making the whole need of raising property change notification redundant.
```

Comment:
```suggestion
It enables leaving the actual _model_ plain and immutable records, making the whole need of raising property change notification redundant.
```
Even if here you are speaking about MVU in general, I would really make clear that records are the way to go for immutability (and feeds).